src/components/FlexContainer.js

```diff
@@ -105,4 +117,5 @@ FlexContainer.defaultProps = {
   alignVertical: 'start',
   gapHorizontal: undefined,
   gapVertical: undefined,
+  stretched: false,
```

Comment:
Should the direction be in the props?

Reply:
Direction is already in die flex container props. You dont need to specify the stretch direction because it'll always be the opposite of the container direction